saleor/graphql/checkout/types.py

```diff
@@ -8,11 +8,11 @@
 from ...core.taxes import display_gross_prices, zero_taxed_money
 from ...extensions.manager import get_extensions_manager
 from ..core.connection import CountableDjangoObjectType
-from ..core.resolvers import resolve_meta, resolve_private_meta
-from ..core.types.meta import MetadataObjectType
 from ..core.types.money import TaxedMoney
 from ..decorators import permission_required
 from ..giftcard.types import GiftCard
+from ..meta.deprecated.resolvers import resolve_meta, resolve_private_meta
```

Comment:
Are these imports used here?

Reply:
Yes, in deprecated part of Checkout type.